### PR TITLE
Add unicode property escapes to regular expressions.

### PR DESCRIPTION
**Pull request by jpmedley**
https://chromiumdash.appspot.com/commits?commit=6f890fd5b212eb441a4b87364922d5286fdc505e

### CODE REVIEW

File: javascript/builtins/RegExp.json [hunk 1]
```diff
@@ -939,6 +939,60 @@
             }
           }
         },
+        "property_escapes": {
+          "__compat": {
+            "description": "property escapes",
```
Comment (Elchi3) [inline javascript/builtins/RegExp.json]:
```suggestion
            "description": "Unicode property escapes (<code>\p{...}</code>)",
```

File: javascript/builtins/RegExp.json [hunk 2]
```diff
@@ -939,6 +939,60 @@
...
+                "version_added": "64"
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": null
```
Comment (Elchi3) [inline javascript/builtins/RegExp.json]:
```suggestion
                "version_added": false
```

File: javascript/builtins/RegExp.json [hunk 3]
```diff
@@ -939,6 +939,60 @@
...
+              },
+              "firefox_android": {
+                "version_added": null
```
Comment (Elchi3) [inline javascript/builtins/RegExp.json]:
```suggestion
                "version_added": false
```

**Comment by jpmedley** [comment]
PTAL

**Pull request merged**